scribus/ui/propertywidget_opentypefontfeatures.cpp

```diff
@@ -43,16 +43,70 @@ void PropertyWidget_OpenTypeFontFeatures::showFontFeatures(QString s)
 {
 	if (!m_ScMW || m_ScMW->scriptIsRunning())
 		return;
-	fontfeatures->setText(s);
```

Comment:
I found that this function is called from `ScribusMainWindow::setItemFontFeatures` which is not used or called anywhere. Is it ok if i removed the whole function ?!